src/script/view_model/LoadingViewModel.js

```diff
@@ -18,23 +18,23 @@
  */
 
 import {t} from 'utils/LocalizerUtil';
+import ko from 'knockout';
 
-window.z = window.z || {};
-window.z.viewModel = z.viewModel || {};
-
-z.viewModel.LoadingViewModel = class LoadingViewModel {
-  constructor(mainViewModel, repositories) {
-    this.elementId = 'loading-screen';
-    this.userRepository = repositories.user;
+export class LoadingViewModel {
+  constructor(userObservable) {
```

Comment:
This class just needs the user (to display the user name in the message).
This can be optimized further by only injecting the user name but at the time it is instantiated, the user is undefined.

Will see if I can find a better option